blocks/product-gallery-block/features/product-gallery/default.jsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { useFusionContext, useComponentContext } from "fusion:context";
+import { RESIZER_APP_VERSION, RESIZER_URL } from "fusion:environment";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+
+import { Carousel, Icon, Image } from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-product-gallery";
+
+export function ProductGalleryDisplay(props) {
+	const { arcSite, data, id, isFeaturedImageEnabled, resizerAppVersion, resizerURL } = props;
```

Comment:
Small code consistency note - we favour unpacking the arguments over destructing them later

```suggestion
export function ProductGalleryDisplay({ arcSite, data, id, isFeaturedImageEnabled, resizerAppVersion, resizerURL }) {
```

Reply:
ok will do

Reply:
updated